### ISSUE TITLE
[Feature request] Combined image/gif/video slideshow with shuffle

### ISSUE THREAD

**Issue by Obsidiman01** [hardcore-sushi/DroidFS#251]
This is a feature that exists in Simple Gallery Pro which I like using, but it doesn't exist in DroidFS. There's an option to shuffle videos, and there's a slideshow for images & gifs, but there isn't a combined mode. Also, I don't think the image slideshow has a shuffle mode, which is another topic on its own. Maybe there could be an option to start a media slideshow, and you could choose which types of media are included via check boxes, that way you can still do video only/image only slideshows. 

I've tried using the "expose encrypted folders" option to open folders in simple gallery pro, but that app doesn't seem to have the ability to view the exposed files, at least not in a way that allows you to play a slideshow of the contents.

**Comment by hardcore-sushi** [comment]
Yes the current slideshow feature is very basic. It could be improved a lot.
